### ISSUE TITLE
Unable to Load application due to yarn package integrity

### ISSUE THREAD

**Issue by isheebo** [ledermann/docker-rails-base#310]
When I try running a built image, I run into this error.

```sh
Puma starting in single mode...
* Version 4.3.5 (ruby 2.7.1-p83), codename: Mysterious Traveller
* Min threads: 5, max threads: 5
* Environment: development
sh: yarn: not found


========================================
  Your Yarn packages are out of date!
  Please run `yarn install --check-files` to update.
========================================


To disable this check, please change `check_yarn_integrity`
to `false` in your webpacker config file (config/webpacker.yml).





! Unable to load application: SystemExit: exit
```

**Comment by ledermann** [comment]
Hm, the assets are installed during the build stage (using Yarn 1.22.4). Therefore, Yarn is not available in the Final stage. Maybe there is something wrong in your Yarn/Webpacker config. Is your `yarn.lock` up to date? Did you use outdated version of Yarn? 

There are lots of hints here:
https://github.com/rails/webpacker/issues/1374

**Comment by isheebo** [comment]
I actually did not tamper with the generated Webpacker/Yarn configuration. To debug the issue, I have created two apps [testapp](https://github.com/isheebo/testapp) and [barebones](https://github.com/isheebo/barebones). The former uses webpacker=stimulus option and the later is a barebones Rails app with no Webpack dependencies. They all give me the same error.

I am using Yarn 1.22.4, Rails 6.0.3.2 and running this on MacOSx Mojave 10.14.6.  All I did was to add the specified Dockerfile (specified in the README), build it and then try running.
